### ISSUE TITLE
How to turn Skipping events notifications off

### ISSUE THREAD

**Issue by heroesx3** [V4NSH4J/discord-mass-DM-GO#546]
hey. Can someone tell me how to turn off the skipping events notifications in dm on react. Its rly confusing for me

**Comment by V4NSH4J** [comment]
Hi, do not mention the channel id, message id and emoji in your config. 
This way the bot will send messages to everyone who reacted in the server.

**Comment by heroesx3** [comment]
Yes, but my observer token is in multiple servers.
And if i only target one server of them it will always notify me when someone reacts in another server. 
So i thought i can just deactivate the notifications. That the tool only tells me if the token sent the DM successfully or not.

**Comment by V4NSH4J** [comment]
yeah, the tool does not do that as of now. You may remove the print lines from the code as per your purpose or alternatively use an observer token which is only present in 1 server.
